### PR TITLE
Code Review Changes

### PR DESCRIPTION
**Pull request by Sambaba23**
Analysis of the program. - I believe that this program is written in a very great manner. I had a really hard to time finding anything that wasn't written well.
Was the program available in UC Github on time? - Yes, This program was available on time.
Is the program easy for you, as an outsider, to understand? - Yes, it was very easy to understand.
Does the program compile? Yes, this program compiles 
Links to three commits you pushed to your group's main/master branch during Sprint 2.
https://github.com/brennerdanae/CachingInApp/commit/c8f677ae941ed3bca28482533257c7a1cdafb86f
https://github.com/brennerdanae/CachingInApp/commit/323052c84fd789a57e6e48848d414b2b801bac59
https://github.com/brennerdanae/CachingInApp/commit/81c2044a06c1785f20f8257b1ff2a992d368cfdb

### CODE REVIEW

File: src/main/java/com/laborhelper/enterprise/dao/LaborActionDAO.java [hunk 1]
```diff
@@ -17,8 +17,7 @@ public List<LaborAction> fetchActionsByLocation(String locationInfo) throws IOEx
         laborActionRetrofitDAO.getLaborAction(locationInfo);
         Call<List<LaborAction>> allAction = laborActionRetrofitDAO.getLaborAction(locationInfo);
         Response<List<LaborAction>> execute = allAction.execute();
-        List<LaborAction> actions = execute.body();
-        return actions;
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/dao/LaborActionDAO.java:21]: The change was made by the recommendation of the code editor

File: src/main/java/com/laborhelper/enterprise/dao/LaborActionRepository.java [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package com.laborhelper.enterprise.dao;
 
 import com.laborhelper.enterprise.dto.LaborAction;
-import com.sun.xml.bind.v2.runtime.reflect.opt.FieldAccessor_Integer;
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/dao/LaborActionRepository.java:4]: removed unused import

File: src/main/java/com/laborhelper/enterprise/dao/LaborActionSQLDAO.java [hunk 3]
```diff
@@ -25,8 +25,7 @@ public List<LaborAction> fetchAllLaborActions() {
 
     @Override
     public LaborAction save(LaborAction laborAction) {
-        LaborAction createdLaborAction = laborActionRepository.save(laborAction);
-        return createdLaborAction;
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/dao/LaborActionSQLDAO.java:29]: made change recommended by IntelliJ.

File: src/main/java/com/laborhelper/enterprise/service/ILaborActionService.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.laborhelper.enterprise.dto.LaborAction;
+
+import java.util.List;
+
+public interface ILaborActionService {
+    LaborAction fetchlaborActionById(int LaborActionid);
+
+  LaborAction save(LaborAction laborAction) throws Exception;
+
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/service/ILaborActionService.java:11]: created a new service package and created a few classes to help get you guys started.

File: src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package com.laborhelper.enterprise.service;
+
+import com.laborhelper.enterprise.dao.LaborActionDAO;
+import com.laborhelper.enterprise.dto.LaborAction;
+import lombok.NonNull;
+
+import java.util.List;
+public class LaborActionServiceStub {
+
+    @Override
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java:10]: created a new service package and created a few classes to help get you guys started.

File: src/main/java/com/laborhelper/enterprise/service/ILaborActionService.java (hunk 4 above)
Comment (discospiff) [reply src/main/java/com/laborhelper/enterprise/service/ILaborActionService.java:11]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
+package com.laborhelper.enterprise.service;
+
+import com.laborhelper.enterprise.dao.LaborActionDAO;
+import com.laborhelper.enterprise.dto.LaborAction;
+import lombok.NonNull;
+
+import java.util.List;
+public class LaborActionServiceStub {
```
Comment (discospiff) [inline src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java:8]: This should implement the interface created, above.

File: src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java (hunk 5 above)
Comment (discospiff) [reply src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java:10]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

**Comment by discospiff** [comment]
From this review:
- IDE recommended changes: you can do this separately, if desired, without merging the branch.
- Partially complete features: only complete features, which meet the team's Definition of Done, should be merged to main.  So, if these are desired, keep the branch open and merge them when they are complete.

**Pull request closed without merging**